pkgs/top-level/all-packages.nix

```diff
@@ -15183,17 +15183,8 @@ with pkgs;
   eiskaltdcpp = libsForQt5.callPackage ../applications/networking/p2p/eiskaltdcpp { };
 
   qemu = callPackage ../applications/virtualization/qemu {
-    inherit (darwin.apple_sdk_12_3.frameworks) CoreServices Cocoa Hypervisor Kernel vmnet;
     inherit (darwin.stubs) rez setfile;
     inherit (darwin) sigtool;
-    stdenv =
-      if stdenv.hostPlatform.isDarwin then
-        overrideSDK stdenv {
```

Comment:
There is another `overrideSDK` in `qemu` that can probably be removed?
https://github.com/NixOS/nixpkgs/blob/7dd2a130125087149391057d00c8927fa73da18e/pkgs/applications/virtualization/qemu/default.nix#L60-L68

Reply:
removed in https://github.com/NixOS/nixpkgs/pull/367764